pynopegl-utils/pynopegl_utils/export.py

```diff
@@ -153,7 +153,7 @@ def _get_scene(**cfg_overrides):
         if "error" in ret:
             print(ret["error"])
             return None
-        return ret
+        return ret["ret"]
```

Comment:
nit: ret["ret"] fields a bit weird, would it make sense to rename the local variable `info` ? Same for `odict`.

Reply:
It becomes `query_info.ret` later on, no worry ;)